src/frontends/pytorch/src/op/fake_quantize.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/fake_quantize.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_fake_quantize_per_tensor_affine(const NodeContext& context) {
+    num_inputs_check(context, 5, 5);
+    auto input_node = context.get_input(0);
+    auto scale = std::make_shared<v0::Convert>(context.get_input(1), element::f32);
+    auto zero_point = std::make_shared<v0::Convert>(context.get_input(2), element::f32);
+    auto out_low = context.const_input<int64_t>(3);
+    auto out_high = context.const_input<int64_t>(4);
+    // Calculate levels value - distance between bounds.
+    auto levels = std::abs(out_high - out_low) + 1;
+    auto out_low_const = v0::Constant::create(element::f32, Shape{1}, {out_low});
```

Comment:
Instead of creating constants here, better would be to use initial inputs obtained by `get_input` that way if FakeQantize will be supporting dynamic levels less changes would be needed and also using initial nodes would make graph easier to match with initial pytorch graph

Reply:
Done